script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_6_1.txt

```diff
@@ -1,22 +1,22 @@
 <Page1>
 ...No, I'll wait here.
-It's reckless to walk around right after coming back.
+Walking around right after coming back would make me seem overly on edge. // 帰ってきたばかりで歩き回るのはさすがに落ち着きがない
 I should greet Akiha before I do anything else.
 She's the current head of the Tohno family, after all.
 <Page2>
 I spend some time sitting on the sofa.
 The view of the yard from the window matches what I remember.
 There is a courtyard the size of four tennis courts, and a garden at the far end that seems ridiculous to even measure.
-On the other side of the garden, there are more woods, and I think――――
+On the other side of the garden, there are more woods, and if I'm not mistaken―――― // 庭園の向こうにはさらに林があって、たしか――― would like to reword 'more woods' but not sure how. 
 <Page3>
 "Oh. I thought I'd stop by before leaving, but I didn't know we had a prior guest."
 The sudden voice made me turn around and gasp.
-In front of the door to the hallway that connects to the lobby,
+In front of the door that connects the hallway to the lobby,
```

Comment:
I don't think this is correct - 
> ロビーに通じる 廊下 側の 扉の 前には、

Reads to me as "The door on the side of the room adjoining the (corridor that connects to the lobby)", which is a heinous sentence but means something a bit different. This current version suggests that the door Saiki came thorough is between the hallway and lobby, but we are currently in the living room

Reply:
will revert lol